### PR TITLE
ZURI CHAT-Fix all the footer links and make it responsive

### PR DESCRIPTION
**Pull request by Chisquare7**
Linear Ticket link: https://linear.app/team-brainbox/issue/FRO-75/zuri-chat-fix-all-the-footer-links-and-make-it-responsive

# Fixes Issue/Linear Ticket
**My PR Fixes ID-linear_ticket_number_FRO-75**

# Changes proposed
Worked on fixing the footer links and route properly

## What were you told to do?
ZURI CHAT-Fix all the footer links and make it responsive

## What did you do?
- I added four div container to the footer with className and makes it responsive
- I styled them with CSS properties such as display, flex-direction, hover property, text-decoration e.t.c
- I also styled the setting Home Links with CSS properties of text-decoration
- I styled all the sidebar links with CSS properties of text-decoration
- I remove non-essential links from the footer section
- I moved help, legal, sign-out folder from old src  to new src folder and fix all the import errors.
- I fixed the routing links of the footer section of SettingsHome.jsx and Other section of AdminSidebar.jsx file

# Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/ Videos

Before changes:
footer-before

Download App link before changes:
download-before

Our Blog link before changes:
our blog before

Help link before changes:
help-link before

Brand Guideline link before changes:
Brand guideline before

Signout Link before changes:
signout-before

After changes for all links that were not routed properly:
image

Download App link:
image

Our Blog link:
image

Help link:
image

Brand Guideline link:
image

Sign-out Link:
image

### CODE REVIEW

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx [hunk 1]
```diff
@@ -185,13 +185,6 @@ const AdminSidebar = ({ setModal, openModal }) => {
       <div className={styles.sidebarMenu}>
         <h6 className={styles.sidebarHeading}>OTHERS</h6>
         <div className={styles.sidebarMenuItems}>
-          <Link
-            onClick={() => setModal(!openModal)}
```
Comment (billmal071) [inline packages/main/src/components/admin-sidebar/AdminSidebar.jsx:189]: use this setModal(prev => !prev).

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx [hunk 2]
```diff
@@ -202,7 +195,7 @@ const AdminSidebar = ({ setModal, openModal }) => {
           <Link
             onClick={() => setModal(!openModal)}
             className={styles.sidebarLink}
```
Comment (billmal071) [inline packages/main/src/components/admin-sidebar/AdminSidebar.jsx:197]: same as here.

**Review by billmal071 — CHANGES_REQUESTED** [review]

File: packages/main/src/App.js [hunk 3]
```diff
@@ -87,7 +87,7 @@ const App = () => (
         <Switch>
           {/* <GeneralLoading /> */}
           <Route exact path="/" component={HomePage} />
-          {/* <Route exact path="/signout" component={SignOut} /> */}
+          {/* <Route exact path="/sign-out" component={SignOut} /> */}
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/App.js:92]: Done

File: packages/main/src/pages/static/legal/components/Content.jsx [hunk 4]
```diff
@@ -0,0 +1,213 @@
+import React, { useState } from "react";
+import Style from "../styles/content.module.css";
+import Link from "../../../components/Link";
+// import CustomLinks from "./contentData"
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/components/Content.jsx:4]: Done

File: packages/main/src/pages/static/legal/components/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+import logo from "../assets/zurilogo.svg";
+import facebook from "../assets/facebook.svg";
+import twitter from "../assets/twitter.svg";
+import linkedin from "../assets/linkedin.svg";
+import instagram from "../assets/instagram.svg";
+import { useTranslation } from "react-i18next";
+// import { googleTranslateElementInit } from "../translator";
+
+const Footer = () => {
+  // googleTranslateElementInit();
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/components/Footer.jsx:14]: Done

File: packages/main/src/pages/static/legal/styles/HeaderStyle.module.css [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+  _margin-right: 30px;
+}
+
+.navLinkPricing {
+  _margin-right: 30px;
+}
+
+.navLinkComms {
+  _margin-right: 30px;
+} */
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/styles/HeaderStyle.module.css:42]: Done

File: packages/main/src/pages/static/legal/styles/HeaderStyle.module.css [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+  padding-bottom: 0;
+}
+.navbarNav li span:hover {
+  padding-bottom: 2px;
+  border-bottom: 0.2rem solid #00b87c;
+}
+
+/* .toggle {
+  _margin-right: 12px;
+} */
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/styles/HeaderStyle.module.css:154]: Done

File: packages/main/src/pages/static/legal/styles/HeaderStyle.module.css [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+    _margin-right: 30px;
+  } */
+
+  .search_wrapper2 {
+    margin-right: -10%;
+  }
+
+  /* .toggle {
+    _margin-right: 30px;
+  } */
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/styles/HeaderStyle.module.css:255]: Done

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx (hunk 2 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-sidebar/AdminSidebar.jsx:197]: Done

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx (hunk 1 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-sidebar/AdminSidebar.jsx:189]: Done

File: packages/main/src/pages/static/sign-out/index.jsx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+      .then(res => {
+        // console.log(res)
+      })
+      .catch(err => {
+        console.error(err);
+      })
+      .finally(() => {
+        window.sessionStorage.clear();
+      });
+  });
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/sign-out/index.jsx:33]: Done

**Comment by Chisquare7** [comment]
All changes done

**Comment by JulianaSau** [comment]
Still have some pending changes to effect @Chisquare7

File: packages/main/src/pages/static/sign-out/index.jsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/sign-out/index.jsx]: This has been done...I added the dependency array here (the rough red circle): 
dependency array

File: packages/main/src/pages/static/legal/components/Content.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/components/Content.jsx]: Done also:
image

File: packages/main/src/pages/static/legal/components/Footer.jsx [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+            </ul>
+          </div>
+        </div>
+
+        {/* <div className={styles.bottom_footer}>
+          <p>
+            {" "}
+            &copy; {getYear()} Zuri Chat. {t("footer_rights")}
+          </p>
+        </div> */}
```
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/components/Footer.jsx:171]: I just did this one....I'm sorry about that

File: packages/main/src/pages/static/legal/styles/HeaderStyle.module.css (hunk 7 above)
[the remark this answers is not in the document]
Comment (Chisquare7) [reply packages/main/src/pages/static/legal/styles/HeaderStyle.module.css]: Also Done:
image

File: packages/main/src/components/admin-sidebar/AdminSidebar.jsx (hunk 2 above)
Comment (Chisquare7) [reply packages/main/src/components/admin-sidebar/AdminSidebar.jsx:197]: Done:
image

**Comment by Chisquare7** [comment]
All are done now ma'am

**Pull request merged**